dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -1012,6 +1012,27 @@ void EnableLightDismiss()
             }
         }
 
+        [TestMethod]
+        public void VerifyTeachingTipButtonsNameAutomationProperty()
+        {
+            using (var setup = new TestSetupHelper(new[] { "TeachingTip Tests", "TeachingTip Test" }))
+            {
+                elements = new TeachingTipTestPageElements();
+
+                SetActionButtonContentTo("Small text");
+                SetCloseButtonContent(CloseButtonContentOptions.ShortText);
+
+                OpenTeachingTip();
+
+                var actionButton = FindElement.ById("ActionButton");
+                var closeButton = FindElement.ById("CloseButton");
+
+                Log.Comment("Verify that action and close buttons content presenter text will update the buttons name automation property");
+                Verify.AreEqual(actionButton.Name, "A:Short Text.");
```

Comment:
>"A:Short Text." [](http://example.com/codeflow?start=51&length=15)

Is this not going to be "Small text"?

Reply:
No, it sets the content to "A:Short Text." in the code behind
`getTeachingTip().ActionButtonContent = "A:Short Text.";`